src/components/TodoInfo/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodoInfo';
```

Comment:
Instead of directly exporting from an import, consider importing the component and then exporting it separately for better readability and consistency.